index.html

```diff
@@ -20,160 +20,164 @@
   </head>
 
   <body>
-    <div class="header">
-
-        <a href="#">
-          <img
-            src="images/bikes-for-refugees_logo.jpg"
-            alt="Logo for Bikes for Refugees Scotland"
-            class="header__logo"
-          />
-        </a>
-
-        <div>
-          <ul class="navigation__list">
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Home</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">About Us</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">News & Events</a>
-            </li>
-            <li class="navigation__item">
-              <a class="navigation__link" href="#">Contact Us</a>
-            </li>
-          </ul>
-        </div>
-
-        <button>Donate Now</button>
```

Comment:
Nice work, but I would recommend that if you could change the buttons to an anchor because we link them to another page. mostly we use anchors rather than a button if we only doing HTML code structure